CONTRIBUTING.md

```diff
@@ -1,11 +1,78 @@
+
+### Process for becoming a committer
+
+* Show your experience with the codebase through contributions and engagement on the community channels.
+* Request to become a committer.
+* Have the majority of committers approve you becoming a committer.
+* Your name and email is added to the MAINTAINERS.md file for the project.
+
+### Committer responsibilities
+
+* Monitor email aliases.
+* Monitor Slack (delayed response is perfectly acceptable).
+* Triage GitHub issues and perform pull request reviews for other committers and the community.
+* Make sure that ongoing PRs are moving forward at the right pace or close them.
+* In general continue to be willing to spend at least 25% of ones time working on the project (~1.25 business days per week).
+
+### When does a committer lose committer status?
+
+If a committer is no longer interested or cannot perform the committer duties listed above, they
+should volunteer to be moved to emeritus status. In extreme cases this can also occur by a vote of
+the committers per the voting process below.
+
+## Technical Steering Committee (TSC) member
+
+The Technical Steering Committee (TSC) oversees the overall technical direction of OpenVDB, as defined in the [charter](charter.md).
+
+TSC voting members consist of committers that have been nominated by the committers, with a supermajority of voting members required to have a committer elected to be a TSC voting member. TSC voting members term and succession is defined in the [charter](charter.md).
+
+All meetings of the TSC are open to participation by any member of the OpenVDB community. Meeting times are listed in the [ASWF techincal community calendar](https://lists.aswf.io/g/tac/calendar).
+
+## Current TSC members
 
 * Ken Museth, Chair / Weta
 * Peter Cucka, DreamWorks
 * Jeff Lait, SideFX
 * Nick Avramoussis, Double Negative
 * Dan Bailey, ILM
 
-Information on how to contribute, meeting frequency and times, dial-in numbers, how to contact the TSC, process for nominating additional membership to the TSC and vetting and voting process, etc. will be forthcoming later in November, 2018.
+# Release Process
+
+Project releases will occur on a scheduled basis as agreed to by the TSC.
+
+# Conflict resolution and voting
+
+In general, we prefer that technical issues and maintainer membership are amicably worked out
+between the persons involved. If a dispute cannot be decided independently, the TSC can be
+called in to decide an issue. If the TSC themselves cannot decide an issue, the issue will
+be resolved by voting. The voting process is a simple majority in which each TSC receives one vote.
+
+# Communication
+
+This project, just like all of open source, is a global community. In addition to the Code of Conduct, this project will:
+
+* Keep all communucation on open channels ( mailing list, forums, chat ).
```

Comment:
```suggestion
* Keep all communication on open channels ( mailing list, forums, chat ).
```

Reply:
As discussed in the last meeting, perhaps also worth highlighting that there is room for private communication for sensitive discussions.